apps/passport-client/src/service-worker.ts

```diff
@@ -10,36 +10,57 @@ const impermanentCache = [
   // "/",
   // "/favicon.ico",
   // "/index.html",
-  // "/global-pcdpass.css",
   // "/global-zupass.css",
-  // "/js/index.js",
+  // "/js/index.js"
 ];
 
-async function addResourcesToCache(resources: string[]): Promise<void> {
+const permanentCache = [
+  "/semaphore-artifacts/16.wasm",
+  "/semaphore-artifacts/16.zkey",
+  "/artifacts/zk-eddsa-event-ticket-pcd/circuit.wasm",
+  "/artifacts/zk-eddsa-event-ticket-pcd/circuit.zkey",
+  "/fonts/IBMPlexSans-Regular.ttf",
+  "/fonts/IBMPlexSans-Medium.ttf",
+  "/fonts/IBMPlexSans-Light.ttf",
+  "/fonts/IBMPlexSans-ExtraLight.ttf",
```

Comment:
Since we're changing things here, this would be a good time to remove the TTF fonts from explicit pre-caching. We already pre-fetch the WOFF versions in index.html, and the TTFs are a fallback for browsers that don't support WOFF (and the set of browsers that don't support WOFF and do support other features we rely on is almost certainly empty).

Reply:
I would also not be opposed to caching the WOFF files here too, though leaving it up to @artwyman

Reply:
I have no informed opinion on this.  I added the `woff` files because I saw them fetched in the log.  I'll remove the `ttf` files now that I understand the difference.

I don't know how the `woff` pre-caching in index.html works.  Is it likely to work offline?  That's the most important reason for the pre-caching here.

I also saw warnings in the logs about `woff` files being pre-fetched but not used.  That might be related to the pre-caching in index.html?  Are you sure these are the right fonts we're using?

Reply:
They are - part of the problem is that the app load time is sufficiently long that none of the pre-cached fonts get displayed within "a few seconds". However, we used to have the problem that fonts would not even start to load until after all of our JavaScript had been processed, meaning that we would often see the wrong fonts being used for a fraction of a second, with an ugly jump as the page layout changes to incorporate the fonts as they load.